client/src/constants/lab9/NavBarData.js

```diff
@@ -0,0 +1,40 @@
+export const NavBarData = {
```

Comment:
just put a short doc string saying this is the user's default state when starting the exercise

Reply:
Development/Lab-9 has been merged